src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const element = document.body.children[1];
+const list = element.children;
+
+const sortList = function(value) {
+  return [...list].sort((a, b) =>
+    Number(b.getAttribute('data-salary').slice(1).split(',').join(''))
+      - Number(a.getAttribute('data-salary').slice(1).split(',').join('')));
+};
+
+element.append(...sortList(list));
+
+const getEmployees = function(value) {
```

Comment:
Same here. You just use the list from the upper scope. But you don't use function parameter